### DISCUSSION TITLE
关于审核流程能看到还没到自己的流程。

### DISCUSSION THREAD

**Discussion by thcoffee** [hhyo/Archery#2617] [General]
a提交 b审核 c审核 。   到b了  c就可以看到 但是不能审核，能不能到b了 c看都看不到。

**Comment by LeoQuote** [comment]
https://github.com/hhyo/Archery/blob/3bd21e7f80953f77da12fe4ff98e21ec945424e7/sql/views.py#L345-L347 

这里确实有改进空间, 欢迎 pr 哈
